Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -119,6 +119,7 @@ fetch() {
   else
     if [[ -f "${temporary_path}" ]]
     then
+      # shellcheck disable=SC2153
```

Comment:
Can this get a comment too (just explaining it's not a typo and where it's set)

Reply:
Without the comment `shellcheck disable`, the message shows on `GitHub ACtion CI/syntax`
```
In /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/cmd/vendor-install.sh line 122:
11
      "${HOMEBREW_CURL}" "${curl_args[@]}" -C - "${VENDOR_URL}" -o "${temporary_path}"
12
       ^--------------^ SC2153: Possible misspelling: HOMEBREW_CURL may not be assigned, but HOMEBREW_CURLRC is.
```

Reply:
```suggestion
      # HOMEBREW_CURL is set by brew.sh (and isn't mispelt here)
      # shellcheck disable=SC2153
```